README.md

```diff
@@ -983,11 +983,14 @@ The constructor `PicovoiceManager.Create` will create an instance of the Picovoi
 ```csharp
 using Pv.Unity;
 
-PicovoiceManager _picovoiceManager = new PicovoiceManager(
+string accessKey = "${ACCESS_KEY}"; // AccessKey obtained from Picovoice Console (https://picovoice.ai/console/)
+
+PicovoiceManager _picovoiceManager = new PicovoiceManager.Create(
+                                accessKey
                                 "/path/to/keyword/file.ppn",
-                                () => {},
+                                () => {}, // wakewordCallback
```

Comment:
maybe let's do `OnWakeWordDetected` and `OnInferenceResult` as below instead of these anonymous lambdas